src/plugins/vis_augmenter/public/vega/helpers.ts

```diff
@@ -0,0 +1,324 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import moment from 'moment';
+import { cloneDeep, isEmpty, get } from 'lodash';
+import {
+  OpenSearchDashboardsDatatable,
+  OpenSearchDashboardsDatatableColumn,
+} from '../../../expressions/public';
+import {
+  PointInTimeEvent,
+  PointInTimeEventsVisLayer,
+  VIS_LAYER_COLUMN_TYPE,
+  EVENT_COLOR,
+  EVENT_MARK_SIZE,
+  EVENT_MARK_SIZE_ENLARGED,
+  EVENT_MARK_SHAPE,
+  EVENT_TIMELINE_HEIGHT,
+  HOVER_PARAM,
+} from '../';
+
+export const enableEventsInConfig = (config: { kibana: {} }) => {
```

Comment:
It's used in the `VegaParser` for config related outside of the scope of the vega config itself, including whether or not it is a map being rendered vs. traditional chart. Because this new field is a similar use case and isn't something native to vega, I'm adding a field to it.

In general, we will always need some extra config to process in the vega parser. We can leave this for now. Future changes to change the name / migrate away from this is outside of the scope of this.